node/_utils.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 import { deferred } from "../async/mod.ts";
-import { assert, assertStringIncludes, fail } from "../testing/asserts.ts";
+import { assert, assertStringIncludes, fail } from "../_util/asserts.ts";
```

Comment:
`fail` looks only used by `mustCall` and it doesn't seem used by anywhere. I think we should remove `fail` and `mustCall`

Reply:
In case of `assertStringIncludes`, it's only used from `assertCallbackErrorUncaught` and that seems like an testing utility (only used from tests).

So we don't need to create `assertStringIncludes` in `_util/asserts.ts`, but rather need to move `assertCallbackErrorUncaught` to somewhere like `node/_test_util.ts`.